cmd/tailscale/ui.go

```diff
@@ -839,12 +839,22 @@ func (ui *UI) layoutPeer(gtx layout.Context, sysIns system.Insets, p *UIPeer, us
 				}),
 				layout.Rigid(func(gtx C) D {
 					var addrs []string
+					var ip4, ip6 string
 					for _, addr := range p.Peer.Addresses {
 						if addr.IP.Is6() {
-							// Don't surface IPv6 addresses.
-							continue
+							if ip6 == "" {
+								ip6 = addr.IP.String()
+							}
+						} else {
+							if ip4 == "" {
+								ip4 = addr.IP.String()
+							}
 						}
-						addrs = append(addrs, addr.IP.String())
+					}
+					if ip4 != "" {
+						addrs = append(addrs, ip4)
+					} else {
+						addrs = append(addrs, ip6)
```

Comment:
If you only ever want to display one address, then I suggest avoiding the "addrs" slice altogether. However, it still seems useful to display one of each address type, if available. That is:

```
if ipv4 != "" {
    addrs = append(addrs, ipv4)
}
if ipv6 != "" {
    addrs = append(addrs, ipv6)
}
```

Reply:
I tried displaying both, if present, but I don't think it works very well:
- it puts a wall of numbers on the screen, but
- the IPv6 address cannot be copied (only the first address is) yet is too long to type it in by hand.

Changed to remove the addrs slice, and display ipv4 if present and ipv6 otherwise.

![Screenshot_20210625-171205](https://user-images.githubusercontent.com/1140756/123496334-76f19f80-d5dc-11eb-857d-fb732674f24c.png)
![Screenshot_20210625-171042](https://user-images.githubusercontent.com/1140756/123496338-7c4eea00-d5dc-11eb-8680-eb666ae1b87d.png)